src/main/kotlin/org/rust/lang/core/macros/errors/GetMacroExpansionError.kt

```diff
@@ -83,6 +89,11 @@ sealed class GetMacroExpansionError {
                 "(maybe it's not provided for your platform by IntelliJ-Rust)"
             ProcMacroExpansionError.ProcMacroExpansionIsDisabled -> "procedural macro expansion is not enabled"
         }
+        ModDataNotFound -> "can't find ModData for containing mod of the macro call"
+        NoMacroIndex -> "can't find macro index of the macro call"
+        ExpansionNameNotFound -> "expansion name not found"
+        ExpansionFileNotFound -> "expansion file not found"
+        InconsistentExpansionCacheAndVfs -> "inconsistent state: expansion file not found, but cache has valid expansion"
```

Comment:
1. I'd prefix these messages with "internal error: " because they are not informative for a user. The only thing that a user can do is attaching the message text to an issue.
2. Some errors are not used now, we can get rid of them. 
3. If I understand it right, `ExpansionFileNotFound` can mean that the expansion task is not finished yet. We could use a more user-friendly message, for example "the macro is not yet expanded".